plugins/backend-ekg/src/Cardano/BM/Backend/EKGView.lhs

```diff
@@ -349,11 +346,11 @@ spawnDispatcher config evqueue _sbtrace ekgtrace =
             ()
             (\_ -> pure ())
 
-    processEKGView obj@(LogObject _ _ _) _ = do
-        obj' <- testSubTrace config ("#ekgview." <> lo2name obj) obj
+    processEKGView obj@(LogObject loname0 _ _) _ = do
```

Comment:
loName' ?

Reply:
`loName` returns the first field of LogObject, or we can pattern match as done here.

Reply:
directly accessed from LogObject